aea/cli/common.py

```diff
@@ -40,7 +39,7 @@
 logger = logging.getLogger("aea")
 logger = default_logging_config(logger)
 
-DEFAULT_REGISTRY_PATH = str(Path(aea.AEA_DIR, "..", "packages").resolve())
+DEFAULT_REGISTRY_PATH = str(Path("..", "packages"))
```

Comment:
please revert it to the previous expression

Reply:
This is the one I have on my local branch. I merged develop.